force-app/main/default/lwc/notificationsTitle/notificationsTitle.html

```diff
@@ -9,12 +9,7 @@ <h2 class="titleName">{notificationHeaderText}</h2>
                     </a>
                 </span>
             </template>
-            <button onclick={closePanel} class="slds-button slds-button_icon slds-m-left_small slds-p-around_small">
```

Comment:
@amityadav2026 same question here...what scenario did this not work for? I've tested this in sandboxes, dev orgs, and production orgs, they all seem to work...so would love to learn more about exactly what site / url structure combination (or other factors) that caused this to not work?

Reply:
@gtabboud for this part I have used source code my in my org I haven't installed package it was throwing error for svg asset not found <use xlink:href="/assets/icons/utility-sprite/svg/symbols.svg#close"></use> this part